internal/importers/base/importer_test.go

```diff
@@ -146,11 +146,10 @@ func TestBaseImporter_ImportProjects(t *testing.T) {
 		importertest.TestCase
 		projects []ImportedPackage
 	}{
-		"tag constraints are ignored": {
+		"tag constraints are skipped": {
 			importertest.TestCase{
-				WantConstraint: "*",
-				WantVersion:    importertest.Beta1Tag,
-				WantRevision:   importertest.Beta1Rev,
+				WantConstraint: "",
```

Comment:
I'm 99% sure that you can remove all the `WantConstraint: ""` in the testcases and the test will work the same. If that works, let's remove these lines from all the testcases.